packages/ui/src/2_molecules/Dialog/components/DialogHeader.module.css

```
@@ -0,0 +1,11 @@
+.container {
+  @apply px-6 py-3.5 relative bg-dialog-header flex
+   flex-row justify-between items-center space-x-4;
+   @screen sm {
+     @apply rounded-t;
+   }
+}
+
+.title {
+  @apply text-white font-bold text-sm;
```

```suggestion
  @apply text-gray-10 font-bold text-sm;
```
According to the Figma it should be `gray-10`.